tensorflow/core/kernels/matmul_op.cc

```diff
@@ -602,7 +603,7 @@ TF_CALL_float(REGISTER_CPU_EIGEN);
 TF_CALL_complex64(REGISTER_CPU);
 TF_CALL_complex128(REGISTER_CPU);
 TF_CALL_double(REGISTER_CPU);
-#else  // INTEL_MKL_DNN_ONLY
+#else   // INTEL_MKL_DNN_ONLY
```

Comment:
The original code already has two spaces between `else` and `//`. Is there a reason for the third space?

Reply:
@penpornk `clang-format` prompted me to add this additional space. I can remove it if you think it is superfluous.

Reply:
Expanding the code, I understand why now. It's to align with <code>#endif &nbsp;// INTEL_MKL_DNN_ONLY</code> (Github didn't show that line in the changes). Please keep it.

Reply:
This line is actually failing our `clang-format` check. Please remove the third space.
```suggestion
#else  // INTEL_MKL_DNN_ONLY
```